src/_gettsim/parameters/erwerbsm_rente.yaml

```diff
@@ -0,0 +1,128 @@
+---
+altersgrenze_abschlagsfrei:
+  name:
+    de: >-
+      Altersgrenze bis zu der man die Erwerbsminderungsrente ohne Abzüge in
+      Anspruch nehmen kann
+    en: null
+  description:
+    de: Bis 2012 immer 63 ab 2012 Anhebung bis 65
+    en: null
+  unit: year
+  reference: >-
+    § 77 SGB VI Zugangsfaktor:
+    https://sozialversicherung-kompetent.de/rentenversicherung/
+    leistungsrecht/295-erwerbsminderungsrente-und-rentenabschlaege.html
+  type: piecewise_linear
+  2001-01-01:
```

Comment:
We chose to start the implementation in 2001 because the Erwerbsminderungsrente was introduced in a reform that year, adding the system before that (Berufsunfähigkeitsrente) would be a task for a new PR in my opinion.

Not sure if we should maybe add a comment about this and add a `dates_active` decorator to the function `erwerbsm_rente_m`?

Reply:
Yes, please. Otherwise we'll have a wrong policy environment pre-2001